packages/components/tailwind.config.js

```diff
@@ -44,6 +45,9 @@ const config = {
       animation: {
         revealVertical: 'revealVertical 400ms forwards cubic-bezier(0, 1, 0.25, 1)',
       },
+      opacity: {
+        15: '.15',
+      },
```

Comment:
Do we need this? [Seems to be included in tailwind already](https://tailwindcss.com/docs/opacity).

Reply:
when I removed this. when I tried `bg-success/15` it did not register the 15% opacity at first. that's why I put those lines.